src/Leapyearcalculator.java

```diff
@@ -0,0 +1,19 @@
+public class Leapyearcalculator {
+    public static void main(String[] args){
+        System.out.println(isleapyear(2001));
+    }
+    public static boolean isleapyear(int year)
+    {
+        if(year>=1 && year<=9999) {
+            if ((year % 4 == 0) && (year % 100 == 0) && (year % 400 == 0)) {
```

Comment:
Why do we need `(year % 4 == 0)` and `(year % 400 == 0)`?

Reply:
I misunderstood the concept.Now,
I have changed the code and 
pushed to the develop branch